app/konnect/network.md

```diff
@@ -62,3 +62,5 @@ for the following parameters:
     cluster_telemetry_endpoint = example.us.tp0.konghq.com:443
     cluster_telemetry_server_name = example.us.tp0.konghq.com
     ```
+{:.note}
```

Comment:
@acgoldsmith I think to fix the formatting, you might just need an extra line between the note and the code block above. 

Also, I'm worried about the note getting buried, would this fit better after the bulleted hostname list? So between the bulleted list and the codeblock above.